--- meta.js.orig	2019-07-12 13:08:08.684983673 +0000
+++ meta.js	2019-07-12 13:07:43.956146610 +0000
@@ -85,6 +85,7 @@
     {name: "mIRC", mime: "text/mirc", mode: "mirc"},
     {name: "MariaDB SQL", mime: "text/x-mariadb", mode: "sql"},
     {name: "Mathematica", mime: "text/x-mathematica", mode: "mathematica", ext: ["m", "nb"]},
+    {name: "Minispec", mime: "text/x-minispec", mode: "minispec", ext: ["ms"], alias: ["ms"]},
     {name: "Modelica", mime: "text/x-modelica", mode: "modelica", ext: ["mo"]},
     {name: "MUMPS", mime: "text/x-mumps", mode: "mumps", ext: ["mps"]},
     {name: "MS SQL", mime: "text/x-mssql", mode: "sql"},
